keyboards/matrix/noah/noah.h

```diff
@@ -48,6 +48,20 @@
     {k40, k41, k42,KC_NO,k43,KC_NO,KC_NO, k45, k46, k47, k48, k1e, k2e, k3e, k0f}  \
 }
 
+#define LAYOUT_iso_uk( \
```

Comment:
This layout seems to match
```suggestion
#define LAYOUT_65_iso_blocker( \
```
which can be added to rules.mk as community layout support:
```
LAYOUTS = 65_iso_blocker
```